pivot/templates/base.html

```diff
@@ -12,7 +12,7 @@
         {% include "head.html" %}
     </head>
 
-<body>
+<body role="application">
```

Comment:
Is role="application" necessary? We generally don't use 'application' because of its limitation. That said, my knowledge could be outdated. Can you share your thoughts?

Reply:
This was just something I came across when looking for guides on getting handlebars to work with aria-live. It apparently helps integrate aria with dynamic webpages. It didn't change anything, but It was listed as a best practice for web applications with dynamic content so I left it in. Apparently it's my understanding that was outdated though. A survey of more recent resources say not to use the application role on body, I'll remove it.